src/ucp/core/ucp_worker.h

```diff
@@ -277,6 +277,7 @@ typedef struct ucp_worker {
  */
 typedef struct ucp_worker_err_handle_arg {
     ucp_ep_h         ucp_ep;
+    ucs_time_t       time_stamp;
```

Comment:
maybe set "min_time" to `ucs_get_time() + worker->context->config.ext.defer_err_handling_time` when this callback is created?